redash/authentication/account.py

```diff
@@ -46,7 +46,7 @@ def send_verify_email(user, org):
     context = {"user": user, "verify_url": verify_link_for_user(user)}
     html_content = render_template("emails/verify.html", **context)
     text_content = render_template("emails/verify.txt", **context)
-    subject = "{}, please verify your email address".format(user.name)
+    subject = _("{}, please verify your email address").format(user.name)
```

Comment:
This works fine? String substitution is sometimes tricky

Reply:
yes it does, I tried the version here in the PR for inviting myself on staging (not exactly this string substitution but the invitation one ) and it worked
<img width="533" alt="Screenshot 2024-12-10 at 12 52 19" src="https://github.com/user-attachments/assets/75933383-1791-4390-bf3e-fda7509d0c74">

Reply:
thanks